eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -70,11 +70,6 @@ jobs:
           pip install doc-warden==$(DocWardenVersion)
           ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
         displayName: "Verify Readmes"
-      - task: UseDotNet@2
-        displayName: "Use .NET Core runtime $(DotNetCoreRuntimeVersion)"
```

Comment:
Are we using these properties anywhere else? If not we might want to remove them from the globals.yml file now.

Reply:
Lets get this PR in ASAP to unblock the sdk roll-forward issue, so we can follow-up in another PR about cleaning up the variables.